src/items/repositories/item.repository.ts

```diff
@@ -85,12 +88,12 @@ export class ItemRepository extends Repository<Item> {
       .getOne();
   }
 
-  async listAnItem(itemId: string, price: number): Promise<void> {
+  async listAnItem(itemId: string, listId: number, price: number): Promise<void> {
     await this.update(
       {
         itemId,
       },
-      { price: String(price), status: ItemStatus.Listed }
+      { price: String(price), listId, status: ItemStatus.Listed }
```

Comment:
I think that we need to clear up the listId when buying an nft.. isn't that right?